data-sync-service/src/l1Ingestion/l1Ingestion.service.ts

```diff
@@ -990,4 +990,84 @@ export class L1IngestionService {
   //     },
   //   }
   // }
+  async getDepositList(address, offset, limit) {
+    const result = []
+    const deposits = await this.txnL1ToL2Repository.findAndCount({
+      where: { from: address },
+      order: { queue_index: 'DESC' },
+      skip: offset,
+      take: limit,
+    });
+    const list = deposits[0];
```

Comment:
Result from db may be null, if deposits is not array, program will report an error
if(deposit?.length > 0)
...